pkg/kudoctl/cmd/init/crds.go

```diff
@@ -35,25 +38,50 @@ func installCrds(client apiextensionsclient.Interface) error {
 func installOperator(client v1beta1.CustomResourceDefinitionsGetter) error {
 	o := generateOperator()
 	_, err := client.CustomResourceDefinitions().Create(o)
-	return err
+	if !isAlreadyExistsError(err) {
```

Comment:
I had hard time understanding these inverted conditions, I guess more natural flow for me would be - for this and the other methods

```go
if isAlreadyExistsError(err) {
		clog.V(4).Printf("crd %v already exists", o.Name)
		return nil
	}

	return err
```

Reply:
yeah... this code evolved from error handling that was repeated and had 3 returns (is statuserror and 1) is already exits or 2) is not already exist and 3) none status error.
now that I've pulled out this function... your comment makes sense... I'll refactor prior to merge.  thanks!!